hack/helloworld.sh

```diff
@@ -1,33 +1,38 @@
 #!/usr/bin/env bash
 
-NAME=hello
-TARGET=${USER:-world}
+set -Eeuo pipefail
+
+# shellcheck disable=SC1091,SC1090
+source "$(dirname "${BASH_SOURCE[0]}")/lib/__sources__.bash"
+
+NAME="${NAME:-hello}"
+TARGET="${USER:-world}"
 
 # Create a sample Knative Service
 cat <<EOF | kubectl apply -f -
 apiVersion: serving.knative.dev/v1alpha1
 kind: Service
 metadata:
-  name: $NAME
+  name: ${NAME}
 spec:
   template:
     spec:
       containers:
         - image: gcr.io/knative-samples/helloworld-go
           env:
             - name: TARGET
-              value: $TARGET
+              value: ${TARGET}
+          readinessProbe:
+            httpGet:
+              path: /
 EOF
 
 # Wait for the Knative Service to be ready
-while output=$(kubectl get ksvc $NAME); do
-  echo "$output"
-  echo $output | grep True >/dev/null && break
-  sleep 2
-done
+timeout 100 "[[ \$(kubectl get ksvc ${NAME} -o \
+jsonpath='{.status.conditions[?(@.type==\"Ready\")].status}') != 'True' ]]"
```

Comment:
Why not

```
kubectl wait --timeout=5m --for=Ready=True
```

? I suppose we could make use of that more often throughout our scripts.

Reply:
I basically changed that to test the new impl of timeout